OrderApi/Migrations/OrdersContextModelSnapshot.cs

```diff
@@ -0,0 +1,83 @@
+﻿// <auto-generated />
```

Comment:
This file will be auto-generated for us when we create our first migration. So we should leave this one too and run our own migration. If it doesn't work, we can replace this back in.